docker-compose.yaml

```diff
@@ -55,6 +60,10 @@ services:
           - 8000:8000
         command: bash -c "python manage.py migrate && python manage.py bootstrap && python manage.py runserver 0.0.0.0:8000"
     client:
+        labels:
+          - "traefik.enable=true"
```

Comment:
Does this line mean that traefik will proxy the container's port?

Reply:
Yes, the labels from line 64-66 will tell traffic that this service to proxy `muscle.local` to this service (which happens to run on port `3000`)